src/main/webapp/components/workspaces/filter-workspaces.js

```diff
@@ -49,12 +51,15 @@ const FilterWorkspaces = ({ onFilter, userAttributes }) => {
 
   return (
     <React.Fragment>
-      <List component="nav">
+      <List component="nav" style={{ maxWidth: '14%' }}>
```

Comment:
:pencil2: this seems like an arbitrary number - wonder if we could do something like `width: fit-content`